src/util-logopenfile.c

```diff
@@ -26,6 +26,7 @@
 #include <sys/types.h>
 #include <sys/socket.h>
 #include <sys/un.h>
+#include <libgen.h>
```

Comment:
woops, an earlier implementation used 'dirname', and therefore needed that include. I must have forgot to remove it. I'll fix it and resubmit.